app/static/credits_style.css

```diff
@@ -0,0 +1,34 @@
+body {
```

Comment:
Please reformat the CSS file using CSS Beautifier of some kind :)

Reply:
fixed :)